lib/outdated.js

```diff
@@ -48,6 +48,12 @@ function outdated (args, silent, cb) {
   outdated_(args, dir, {}, 0, function (er, list) {
     if (!list) list = []
     if (er || silent || list.length === 0) return cb(er, list)
+    list.sort(function(a, b) {
+      var aa = a[1].toLowerCase()
+        , bb = b[1].toLowerCase()
+      return aa === bb ? 0
```

Comment:
@othiym23 nvm. I was wrong. `a` and `b` are arrays in the form,

    [ '/home/thefourtheye/git/npm/test/tap/outdated',
      'underscore',
      '1.3.1',
      '1.3.1',
      '1.5.1',
      '1.3.1' ]

So, the code works fine only I guess.